src/classes/EscalationTeamTrigger_Test.cls

```diff
@@ -0,0 +1,480 @@
+/**
+* Test for the escalation team trigger
+*
+* @author Preeti (ppreeti@redhat.com)
+*/
+
+@IsTest
+public with sharing class EscalationTeamTrigger_Test {
```

Comment:
setUniqueContactTypeMember_onAnEscalationTeamUpdateTest and other methods with such functionality are present in test class